flake.nix

```diff
@@ -3,7 +3,7 @@
     common.url = "github:nammayatri/common";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/47f4acaf698a9f4f6b167ca905d76348a9495767";
```

Comment:
Do not change this file. Use this command:
nix flake lock --update-input shared-kernel --override-input shared-kernel github:nammayatri/shared-kernel/47f4acaf698a9f4f6b167ca905d76348a9495767